praxis-isncsci/ui
# Key points diagram

Closes #184 

## Problem

- Render a key points diagram
- The element needs to be fixed in the middle of the screen

Dynamically changing the colors based on values will be done in a different story (#185).

## Solution

- Added SVG diagram
- Used position fixed to position the diagram in the middle of the screen and always visible

## Testing

- [x] The key points diagram is visible in the middle of the interface

<details>
  <summary>Test case</summary>

  1. Navigate to the [web app **Storybook** story](https://64f8d7c6e093108e99084a70-kqduukbnui.chromatic.com/?path=/story/app-app--primary)
  2. Confirm that the diagram is visible
  3. Scroll up and down
  4. Confirm the diagram remains fixed in the middle of the interface

</details>
